### PR TITLE
using variable for deployment_type

### PR DESCRIPTION
**Pull request by jfcoz**
https://github.com/openshift/openshift-ansible/issues/1632

### CODE REVIEW

File: playbooks/adhoc/metrics_setup/playbooks/install.yml [hunk 1]
```diff
@@ -9,8 +9,11 @@
     lineinfile: "state=present dest=/etc/origin/master/master-config.yaml regexp='^\ \ metricsPublicURL' insertbefore='^\ \ publicURL' line='\ \ metricsPublicURL: {{ metrics_public_url }}'"
 
   - name: "Restart master service"
-    service: name=atomic-openshift-master state=restarted
+    service: name={{deployment_type}}-master-api state=restarted
```
Comment (sdodson) [inline playbooks/adhoc/metrics_setup/playbooks/install.yml]: {{ openshift.common.service_type }}  is what we use for this

**Comment by detiber** [comment]
@sdodson I suspect the need for this will be negated with your upcoming changes. Please close if that is the case.

**Comment by detiber** [comment]
@sdodson can you confirm this is no longer needed?

**Comment by sdodson** [comment]
Yes, no longer necessary.

**Pull request closed without merging**